// @whiskeysockets/baileys/lib/index.js
//=======================================================//
import gradient from "gradient-string"
import makeWASocket from "./Socket/index.js"

const g = {
  green: gradient('#25D366', '#128C7E'),
  sky: gradient('#00D9FF', '#FF00E5'),
  neon: gradient('#39FF14', '#FF00FF'),
  pastel: gradient.pastel,
  credit: gradient('#FF9A00', '#FF5E6C')
}

//console.clear()

//=======================================================//
//                      B A I L E Y S
//=======================================================//
console.log(g.pastel("╔══════════════════════════════════════════════════════════════════╗"))
console.log(g.pastel("║                            BAILEYS MOD                           ║"))
console.log(g.pastel("╚══════════════════════════════════════════════════════════════════╝\n"))

// === LOGO GRANDE (80 col) ===
console.log(g.green(`
██████╗   █████╗ ██╗██╗     ███████╗██╗   ██╗███████╗
██╔══██╗ ██╔══██╗██║██║     ██╔════╝╚██╗ ██╔╝██╔════╝
██████╔╝ ███████║██║██║     █████╗   ╚████╔╝ ███████╗
██╔══██╗ ██╔══██║██║██║     ██╔══╝    ╚██╔╝  ╚════██║
██████╔╝ ██║  ██║██║███████╗███████╗   ██║   ███████║
╚═════╝  ╚═╝  ╚═╝╚═╝╚══════╝╚══════╝   ╚═╝   ╚══════╝
`))
// === INFO ===
console.log(g.neon("Baileys Mod • Multi-Device • Sem Bugs • Estável"))
console.log(g.sky("by BLUX | Estável\n"))

// === REDES ===
console.log(g.pastel("Redes Sociais"))
console.log("• whatssap: 558189728863")

// === CRÉDITOS ===
console.log(g.credit("══════════════════  CRÉDITOS  ══════════════════"))
console.log(g.credit("BLUX • Desenvolvedor Oficial / Open Source Quack"))
console.log(g.credit("whatssap: +55 81 8972-8863"))
console.log(g.credit("══════════════════════════════════════════════════\n"))

// === FOOTER ===
console.log(g.neon("Feito com código limpo, café e dedicação ☕"))
console.log(g.sky("© 2025 BLUX • Estável \n"))

//=======================================================//
export * from "./Defaults/index.js"
export * from "./WABinary/index.js"
export * from "../WAProto/index.js"
export * from "./WAUSync/index.js"
export * from "./Store/index.js"
export * from "./Utils/index.js"
export * from "./Types/index.js"
export * from "./WAM/index.js"
export { makeWASocket }
export default makeWASocket
//=======================================================//
